pages/login.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+
+import { loginFormTestIds } from '@/components/forms/Login';
+import Login, { loginTestIds } from './login';
+
+describe(Login, () => {
+  beforeEach(() => {
+    render(<Login />);
+  });
+
+  it('renders log in text', () => {
```

Comment:
How about removing this space here to make it consistent with the below test case?

```suggestion
  it('renders login text', () => {
```

Reply:
done in https://github.com/tyrro/react-survey/pull/50/commits/ac569c09a1643adb280fe4d6a3705d1c347ab318